inst/examples/AA-AA-000-0000/workflows/2_metrics/cou0007.yaml

```diff
@@ -0,0 +1,83 @@
+meta:
+  Type: Analysis
+  ID: cou0007
+  GroupLevel: Country
+  Abbreviation: TDSC
+  Metric: Treatment Discontinuation Rate
+  Numerator: Subjects Discontinued - Treatment
+  Denominator: Enrolled Subjects
+  Model: Normal Approximation
+  Score: Adjusted Z-Score
+  AnalysisType: binary
+  Threshold: -3,-2,2,3
+  nMinDenominator: 3
+spec:
+  Mapped_SUBJ:
+    subjid:
+      required: true
+      type: character
+    country:
+      required: true
+      type: character
+  Mapped_SDRGCOMP:
+    subjid:
+      required: true
+      type: character
+    sdrgyn:
+      required: true
+      type: character
+    phase:
+      required: true
+      type: character
+steps:
+  - name: ParseThreshold
+    output: vThreshold
+    params:
+      strThreshold: Threshold
+  - name: RunQuery
+    output: Temp_DISCONTINUED
+    params:
+      df: Mapped_SDRGCOMP
+      strQuery: "SELECT * FROM df WHERE sdrgyn = 'N' AND phase = 'Blinded Study Drug Completion'"
+  - name: Input_Rate
+    output: Analysis_Input
+    save: true
```

Comment:
This `save` param does break in-memory workflows because it requires a `lConfig` object that is not specified when we run this locally. we can do this, but i'm curious why we need this `save` param if all of the tables get spit out as a list at the end? can't they just be extracted from that list and used as needed?

Reply:
Like adding a loop in `SaveData` for lists.  That might work a little better...

Reply:
@jwildfire - I think this problem goes away if each analysis df has its own yaml, like the Reporting dfs have. Do we want to go that route?

Reply:
we would no longer need `save` or the list of dfs because each file would just write out the file for that step.